src/NuGet.Protocol.Core.v3/DataClient/DataClient.cs

```diff
@@ -27,6 +27,14 @@ public sealed class DataClient : HttpClient
         private readonly INuGetMessageHandlerProvider[] _modifiers;
```

Comment:
Looks like this field is not used anywhere in the class today. If this is needed for the future, I would consider filing a bug to track this. If not, I will delete it